### PR TITLE
feature: ZENKO-2885 Package zookeeper in solution-base solution

### PR DESCRIPTION
**Pull request by miniscruff**
**What does this PR do, and why do we need it?**

Adds zookeeper to base solution

**Which issue does this PR fix?**

fixes ZENKO-2885

**Special notes for your reviewers**:
Will need to resolve conflicts with Kafka PR when that is merged.
Also fixed a typo

### CODE REVIEW

File: solution-base/kubedb/certs.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```
Comment (miniscruff) [inline solution-base/kubedb/certs.yaml:1]: kubedb cert-manager is behind a paywall now so I do not believe they will update to 1.0 for the version we are using.

File: solution-base/zookeeper/operator.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  template:
+    metadata:
+      labels:
+        name: zookeeper-operator
+    spec:
+      serviceAccountName: zookeeper-operator
+      containers:
+        - name: zookeeper-operator
+          # Replace this with the built image name
+          image: ${ZOOKEEPER_IMAGE_PREFIX}/zookeeper-operator:${ZOOKEEPER_OPERATOR_TAG}
```
Comment (tcarmet) [inline solution-base/zookeeper/operator.yaml]: I saw that we used `KUBEDB_IMAGE_NAME` or `KAFKA_OPERATOR_IMAGE_NAME` (don't know if that was the exact name but the idea was there) should we make one for `zookeeper-operator` as well?

Comment (miniscruff) [reply solution-base/zookeeper/operator.yaml]: Sure, will do.

Comment (miniscruff) [reply solution-base/zookeeper/operator.yaml]: done*

**Review by tcarmet — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]

**Pull request merged**